python/ray/serialization_addons.py

```diff
@@ -0,0 +1,52 @@
+"""
+This module is intended for implementing internal serializers for some
+site packages.
+"""
+
+
+def register_pydantic_serializer(serialization_context):
+    try:
+        import pydantic.fields
+    except ImportError:
+        return
+
+    # Pydantic's Cython validators are not serializable.
+    # https://github.com/cloudpipe/cloudpickle/issues/408
+    serialization_context._register_cloudpickle_serializer(
```

Comment:
What's the reason for using an internal API instead of `ray.util.register_serializer`?

Reply:
The ray.util method doesn't work for Ray client's worker.

Reply:
We should make a unified public API! @AmeerHajAli can you make sure to follow up on this?

Reply:
CC @ijrsvt who is leading these efforts.